services/cinema_hall.py

```diff
@@ -0,0 +1,20 @@
+from django.db.models import QuerySet
+from django.utils.datetime_safe import datetime
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet[CinemaHall]:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(
+        hall_name: datetime,
```

Comment:
The parameter `hall_name` is incorrectly typed as `datetime`. It should be `str` since it represents the name of the cinema hall. Update the type to `str`.